src/runtime/hexagon/hexagon/hexagon_common.cc

```diff
@@ -57,7 +57,10 @@ void HexagonLookupLinkedParam(TVMArgs args, TVMRetValue* rv) {
   std::vector<int64_t> shape_vec{template_tensor->shape,
                                  template_tensor->shape + template_tensor->ndim};
 
-  auto* param_buffer = new HexagonBuffer(static_cast<void*>(opaque_handle));
+  // TODO: get nbytes from shape/dtype
+  size_t nbytes = 2048;
```

Comment:
Hard coded value?

Reply:
Right, wasn't quite sure how to calculate the size so I left a TODO there.  Need to do some calculation on shape and dtype.

Reply:
Done.